plotly_resampler/aggregation/aggregators.py

```diff
@@ -43,61 +46,45 @@ class LTTB(AbstractSeriesAggregator):
     * To aggregate category data with LTTB, your ``pd.Series`` must be of dtype
       'category'. |br|
       **Tip**: if there is an order in your categories, order them that way, LTTB uses
-      the ordered category codes values (se bullet above) to calculate distances and
+      the ordered category codes values (see bullet above) to calculate distances and
       make aggregation decisions.
       .. code::
+        >>> import pandas as pd
         >>> s = pd.Series(["a", "b", "c", "a"])
         >>> cat_type = pd.CategoricalDtype(categories=["b", "c", "a"], ordered=True)
         >>> s_cat = s.astype(cat_type)
 
     """
 
-    def __init__(self, interleave_gaps: bool = True, nan_position="end"):
+    def __init__(self, interleave_gaps: bool = True):
         """
         Parameters
         ----------
         interleave_gaps: bool, optional
             Whether None values should be added when there are gaps / irregularly
             sampled data. A quantile-based approach is used to determine the gaps /
             irregularly sampled data. By default, True.
```

Comment:
Types not necessary & mismatched. Also interleave_gaps should be optional in the type signature.